client/lib/query-manager/index.js

```diff
@@ -104,6 +104,27 @@ export default class QueryManager {
 	}
 
 	/**
+	 * A sorting function that defines the sort order of items under
+	 * consideration of the specified query.
+	 *
+	 * @param  {Array}  keys  Keys to be sorted
+	 * @param  {Array}  items Items by which to sort
+	 * @param  {Object} query Query object
+	 * @return {Array}        Sorted keys
+	 */
+	sort( keys, items, query ) {
+		return keys.sort( ( keyA, keyB ) => {
```

Comment:
Yeah, likewise 😕  Well unfortunately `Array.prototype.sort` is mutating, and I wasn't sure about copying the arg here for clarity vs performance of just mutating the arg. Are you suggesting to copy the arg? I guess it won't hurt too much under the aspect of eternity 😁